### ISSUE TITLE
[protobuf] Check fails

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#3037]
```
Checking...
Making check in .
make[1]: Entering directory '/usr/local/tmp/crew/v3.6.1.3.tar.gz.dir/protobuf-3.6.1.3'
make  check-local
make[2]: Entering directory '/usr/local/tmp/crew/v3.6.1.3.tar.gz.dir/protobuf-3.6.1.3'
Making lib/libgmock.a lib/libgmock_main.a in gmock
/bin/bash: line 0: cd: third_party/googletest/googletest: No such file or directory
make[2]: *** [Makefile:1968: check-local] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/v3.6.1.3.tar.gz.dir/protobuf-3.6.1.3'
make[1]: *** [Makefile:1829: check-am] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/v3.6.1.3.tar.gz.dir/protobuf-3.6.1.3'
make: *** [Makefile:1538: check-recursive] Error 1
protobuf failed to build: `make check` exited with 2
```

**Comment by JL2210** [comment]
That package supposedly needs the Googletest sources to check. You can go ahead and remove the `def self.build` section.

**Comment by uberhacker** [comment]
You mean the `def self.check` section?

**Comment by JL2210** [comment]
Yes. Sorry.

**Comment by cstrouse** [comment]
This was fixed by PR #3041.
